packages/react-core/src/components/Label/Label.tsx

```diff
@@ -90,7 +90,7 @@ export const Label: React.FunctionComponent<LabelProps> = ({
   isCompact = false,
   isEditable = false,
   editableProps,
-  isTruncated = false,
+  isTruncated = true,
```

Comment:
I don't think we need this bool anymore, labels will truncate when they run out of room in their container to display their contents. You can specify a custom truncation length though, so we'll need a prop for that that takes a length and is applied to `.pf-c-label__text` via `style="--pf-c-label__text--MaxWidth: {{ // text max width prop value }}"` - you can see that here https://pf5.patternfly.org/components/label/#filled